bin/export_open_targets.sh

```diff
@@ -89,18 +89,19 @@ listExperimentsToRetrieve | while read -r experimentAccession ; do
     rm -f "$experimentAccession.tmp.json"
   fi 
 done
-rm -rf experiments-exclude.tmp
 
 # Actually exit if the while read loop hasn't exited successfully
 if [ $? -ne 0 ]; then
   echo "OT export failed"
   exit 1
 fi
 
+rm -rf experiments-exclude.tmp
+
 trap - INT TERM EXIT
 
 echo "Successfully fetched and validated evidence, zipping..."
-mv -nv ${destination}.tmp $destination && rm -f ${$destination}.gz && gzip $destination
+mv ${destination}.tmp $destination && gzip $destination
```

Comment:
unless you need $destination for some other purpose,  I would `cat ${destination}.tmp | gzip - ` with maybe some proper output name for gzip. But if you have already tested it and you don't want to go back, I would just leave it.